### DISCUSSION TITLE
Are these features possible?

### DISCUSSION THREAD

**Discussion by yuzuki76** [arianneorpilla/jidoujisho#269] [General]
Hi, would it be possible to have a word search feature for the ttsu reader/mokuro? Jidoujisho is truly incredible and has replaced nearly all of my immersion tools, but I still have to open up an epub reader (Typhon) to look up where a word is used/how many times it appears in a book. 

My favorite feature from the last few updates is the ability to see that I have already made a card in Anki. Would it be possible to then open that card in Anki/the card browser in Anki with that term as the search with one click? Currently I copy the word, open the Anki card browser, paste word, search, and then find the card.

**Comment by arianneorpilla** [comment]
The existing Anki card feature is probably possible as a quick action in the future. I'm currently in hiatus but I'll consider this when I'm not very busy sometime.

As for a search feature, I don't work on TTSU and am only a package consumer, so I don't think I can make word search work as is.

**Comment by brishtibheja** [comment]
@yuzuki76 Can I ask what do you mean by the ability to see card creation completion? Is it the temporary message that shows up on bottom? If you mean that one, it is actually annoying imo. It covers a lot of the screen especially when watching anime. And then sometimes it covers up the part that I am reading from つっ reader. The second feature you suggested was great though. Manual editing will get much more easier.

@lrorpilla could you make that card created message optional when you get time from your work? I already know the card is created when the fields automatically clear. Also consider it for the message that shows up when you lock sentences in card creator.

**Reply by arianneorpilla** [reply]
I'm going to be frank and say I'm not really going to fulfill any non-critical requests for this app for the time being. I'm really busy at the moment and people shouldn't expect anything from me for the near future -- my hiatus is indefinite and I really don't have a lot of free time these days.

I encourage users to fork the application if they have any  requests that are very particular to their individual tastes.

For example, for the change you want, you probably want to comment [this line](https://github.com/lrorpilla/jidoujisho/blob/cbf7cebe6e9f7ef87ab79089b884b4c83066ed2c/yuuna/lib/src/models/app_model.dart#L2117) and compile.

You can search uses of the same function to track down a similar one for locking fields.

**Reply by brishtibheja** [reply]
I'm on github but have no knowledge of programming so I probably wouldn't be able to solve the issue myself. But I thought hiatuses were a common part of this project. I don't know if it is different this time but please consider my issue whenver you get time. This is a great app so I can wait in the meantime.

Edit: I will add this: the card created message that is shown is probably not a really critical issue. But it is also not an issue particular to me imo. In my realme phone, I can change the settings for this. But even then, the minimum time for it that I can select is 10sec.
